### PR TITLE
docs(pr): describe pull request workflow

### LINKED ISSUE #21: Write out PR process
- Write down PR
- Write PR template

### PR DESCRIPTION
**Pull request by vringar**
Closes #21

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.vscode
```
Comment (Ti-ger) [inline .gitignore:1]: nit: Does this really belong into this PR?

File: docs/pull_requests.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## As the Author
+
+1. Write code on a feature branch
+2. Create a Draft PR as soon as you push something
+3. Assign a reviewer whenever you want a review (This can be before the code is ready to merge)
+    1. If your reviewer requests changes, do them or argue why they are not required
+    2. Don't forget to re-request the review, if you are ready for a second round
+4. Once your code is ready for merge/final review, mark the PR by removing the draft
+5. If you have the approval of a reviewer and adressed all comments you are free to merge
+
```
Comment (Ti-ger) [inline docs/pull_requests.md:14]: Not sure if this is the place, but we also talked about a PR-Template/Checklist.
Things that come to my mind:
- Only request reviews for PRs that are: buildable and don't break other code
- Software Architecture checklist (#22)
- Linting/Code Style (maybe we need an issue for that; matter or at least the VSCode subset uses clang-format: https://github.com/project-chip/connectedhomeip/blob/master/docs/VSCODE_DEVELOPMENT.md#visual-studio-code-recommended-settings)

just some thoughts so, feel free to merge ;)

**Review by Ti-ger — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (vringar) [reply .gitignore:1]: It doesn't, sorry. Leaving it in anyway.

File: docs/pull_requests.md (hunk 2 above)
Comment (vringar) [reply docs/pull_requests.md:14]: I think building and code formatting are excellent ideas for automation and not humans.

@Emirhan-Atabay do you want to create an issue for yourself, that describes these two use cases

Comment (vringar) [reply docs/pull_requests.md:14]: Thanks for the reminder @Ti-ger. I've added a basic template that should capture some basic steps.

**Review by Ti-ger — APPROVED** [review]
The file is empty. Is this intended? 

nit: I noticed we name our branches quite different. I named them after the issue, while you named them after the position in the filesystem.

**Pull request merged**